addons/addon-environment-sc-api/packages/environment-sc-workflow-steps/lib/steps/__test__/launch-product.test.js

```diff
@@ -0,0 +1,161 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { iteratee } = require('lodash');
+const WorkflowPayload = require('@aws-ee/workflow-engine/lib/workflow-payload');
+const LaunchProduct = require('../launch-product/launch-product');
+
+describe('LaunchProduct', ()=> {
+      const meta = {};
+      const input = {};
+    
+      const lp = new LaunchProduct({
+        step: { config: {} },
+        workflowPayload: new WorkflowPayload({ meta, input, workflowInstance: { steps: [] } }),
+      });
+
+    // afterEach(() => {
+    //     const resolvedInputParams = [];
+    //     const datetime = 0;
+    //     const datetime2 = 0;
+    // })
+
+    describe('checkNamespace', () => {
+        // it('Static name should be transformed to start with analysis- and end with number string and be unique between calls', async () => {
+        //     // Build 
+        //     const resolvedInputParams = [{Key: 'Namespace', Value: 'staticname'}];
+        //     const resolvedInputParams2 = [{Key: 'Namespace', Value: 'staticname'}];
+        //     const datetime = Date.now();
+        //     const datetime2 = Date.now();
+        //     const originalNamespace = resolvedInputParams[0].Value;
+        //     // const originalNamespace2 = resolvedInputParams2[0].Value;
+    
+        //     // Operate
+        //     const { afterNamespace, index } = await lp.getNamespaceAndIndexIfNecessary(resolvedInputParams, datetime);
+        //     const { afterNamespace2, index2 } = await lp.getNamespaceAndIndexIfNecessary(resolvedInputParams2, datetime2);
+    
+        //     // Check
+        //     await expect(afterNamespace.startsWith('analysis-')).toBeTruthy();
+        //     await expect(afterNamespace).not.toBe(afterNamespace2);
+        //     await expect(afterNamespace).toBe('analysis-' + originalNamespace + `-${datetime}`);
+        // });
```

Comment:
The commented code can be removed I think unless there is a reason we want to keep it around.

Reply:
Yes, I will remove it. Sorry about that!